configuration_controller/response_processor/strategies/response_processing.py

```diff
@@ -0,0 +1,141 @@
+import logging
+from typing import List
+
+from db.db import Session
+from db.models import DBGrant, DBGrantState, DBResponse
+from db.types import GrantStates, RequestTypes
+
+logger = logging.getLogger(__name__)
+
+
+_CBSD_ID = "cbsdId"
```

Comment:
please remove the leading underscore like you did (rightfully and righteously so ;)) above